core/core.go

```diff
@@ -229,11 +229,11 @@ func unquoteConfig(config Config) Config {
 }
 
 func getPubSubTopic(networkID int) string {
-	return fmt.Sprintf("/0x-orders/network/%d/version/1", networkID)
+	return fmt.Sprintf("/0x-orders/network/%d/version/2", networkID)
```

Comment:
@fabioberger I wonder if we really need to bump the version number for the rendezvous string and pubsub topic? The format and types of messages we send to peers has not changed, just one detail about the semantics of which orders are considered valid.

If we don't change the version number, the worst thing that could happen is that some peers will send messages to others that they consider invalid due to the change in semantics. If you receive too many such messages, you will eventually disconnect from that peer. This will should be rare in practice and already can happen anyways due to differing order expiration buffers.

Reply:
Thinking about it again, I agree. Also, there have only been ~10 orders that fit this edge-case on Radar right now. Let's stay on the same version and urge as many people to upgrade their nodes.